messaginginapp/api/messaginginapp.api

```diff
@@ -234,6 +236,21 @@ public final class io/customer/messaginginapp/gist/utilities/ElapsedTimer {
 	public final fun start (Ljava/lang/String;)V
 }
 
+public final class io/customer/messaginginapp/state/EmbeddedMessagesState {
```

Comment:
Do we need to make all of these public? Can we please double check if there's anything that can be kept internal instead?